plugin/pkg/admission/limitranger/admission.go

```diff
@@ -383,6 +375,38 @@ func sum(inputs []api.ResourceList) api.ResourceList {
 	return result
 }
 
+// DefaultLimitRangerActions is the default implementatation of LimitRangerActions.
+type DefaultLimitRangerActions struct{}
+
+// ensure DefaultLimitRangerActions implements the LimitRangerActions interface.
+var _ LimitRangerActions = &DefaultLimitRangerActions{}
+
+// Limit enforces resource requirements of incoming resources against enumerated constraints
+// on the LimitRange.  It may modify the incoming object to apply default resource requirements
+// if not specified, and enumerated on the LimitRange
+func (d *DefaultLimitRangerActions) Limit(limitRange *api.LimitRange, resourceName string, obj runtime.Object) error {
+	switch resourceName {
+	case "pods":
+		return PodLimitFunc(limitRange, obj.(*api.Pod))
+	}
+	return nil
+}
+
+// SupportsAttributes ignores all calls that do not deal with pod resources since that is
+// all this supports now.  Also ignores any call that has a subresource defined.
+func (d *DefaultLimitRangerActions) SupportsAttributes(a admission.Attributes) bool {
+	if a.GetSubresource() != "" {
```

Comment:
are we confident in our downstream use cases that we will not forget to include this line?